components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -57,6 +57,7 @@ class PlacesApi(path: String) : PlacesManager, AutoCloseable {
         // function instead :(
         api = placesApiNew(path)
 
+        println("JONATHANNNNNN WAS HERE")
```

Comment:
nit: I think we should remove @jonalmeida from here XD

Reply:
Oops.